README.md

```diff
@@ -8,34 +8,63 @@ Help on flags:
-  -h, --help                   Show context-sensitive help (also try --help-long and --help-man).
+  -h, --help                 Show context-sensitive help (also try --help-long and --help-man).
       --telemetry.address=":9702"
-                               Address on which to expose metrics.
+                             Address on which to expose metrics.
       --telemetry.endpoint="/metrics"
-                               Path under which to expose metrics.
-      --interface=INTERFACE    Interface to search for Homeplug devices.
-      --destaddr=00B052000001  Destination MAC address for Homeplug devices.
-      --log.level="info"       Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal]
-      --log.format="logger:stderr"
-                               Set the log target and format. Example: "logger:syslog?appname=bob&local=7" or "logger:stdout?json=true"
-      --version                Show application version.
+                             Path under which to expose metrics.
+      --interface=INTERFACE  Interface to search for Homeplug devices.
+      --destaddr=local       Destination MAC address for Homeplug devices. Accepts 'local', 'all', and 'broadcast' as aliases.
+      --log.level=info       Only log messages with the given severity or above. One of: [debug, info, warn, error]
+      --log.format=logfmt    Output format of log messages. One of: [logfmt, json]
+      --version              Show application version.
 ```
 
-Tested with TP-Link TL-PA4010, but should work with any device that supports HomePlug AV or better.
+Tested with TP-Link TL-PA4010, but should work with any device using a Qualcomm
+Atheros chipset such as QCA6410, QCA7000, and QCA7420.
 
-The default destination MAC address will elicit a response from any HomePlug devices on the local Layer 2 network segment.
-This will NOT find devices on the far side of a Power Line bridge. If you know the MAC address of a device, including a
-device on the far side of a Power Line bridge, you may override the destination address.
+The default destination MAC address (`local`) will elicit a response from any
+HomePlug devices on the local Layer 2 network segment.  This will NOT find
+devices on the far side of a Power Line bridge. If you know the MAC address of
+a device, including a device on the far side of a Power Line bridge, you may
+override the destination address.
+
+The `destaddr` parameter accepts these special values:
+
+`all`
+    Same as `broadcast`.
+
+`broadcast`
+    A synonym for the Ethernet broadcast address, `FF:FF:FF:FF:FF:FF`.
+    All devices, whether local, remote or foreign recognize messages sent to this address.
+
+`local`
+    A synonym for the Qualcomm Atheros vendor specific Local Management Address (LMA), `00:B0:52:00:00:01`.
+    All local Atheros devices recognize this address but remote and foreign devices do not.
+
+A remote device is any device at the far end of a powerline connection.
```

Comment:
Should "powerline" be written as "Power Line" as it is elsewhere in the document?

Reply:
Good point, I think I will just use PLC and adjust a few other inconsistencies this made me reallise

Reply:
Hmm, PLC says "programmable logic controller" to my brain, as I've worked with such devices in the past. I'd be wary of abbreviating it as such.

Reply:
Yeah, that can be a bit confusing, but it seems to be the term used. Also, it helps differentiate when talking about "power lines" in general